app/controllers/users_controller.rb

```diff
@@ -1,18 +1,53 @@
 class UsersController < ApplicationController
-  before_action :set_user
+  before_action :set_user, except: [:index, :new, :create]
+  after_action :verify_authorized, except: :index
+  after_action :verify_policy_scoped, only: :index
+
+  def index
+    @users = policy_scope(User)
+  end
+
+  def show
+    authorize @user
+  end
+
+  def new
+    @user = User.new
+    authorize @user
+  end
+
+  def create
+    @user = User.new user_params
+
+    authorize @user
+
+    if @user.save
+      current_user.organisations.each do |organisation|
```

Comment:
Absolutely. This is a dirty hack to force new Users to belong to an Organisation.
The process for creating a New User will be to create it in an Organisation ie nesting like /organisations/:organisation_id/users/new so that the Org is a User choice